community/xf86-video-fbdev/patches/gcc8.patch

```diff
@@ -0,0 +1,11 @@
+--- src/fbdev.c	2018-05-11 21:15:36.650039778 +0200
++++ src/fbdev.c	2018-05-11 21:17:27.063021717 +0200
+@@ -1008,7 +1008,7 @@
+ 	fPtr->CloseScreen = pScreen->CloseScreen;
+ 	pScreen->CloseScreen = FBDevCloseScreen;
+ 
+-#if XV
++#ifdef XV
+ 	{
+ 	    XF86VideoAdaptorPtr *ptr;
+ 
```

Comment:
Can be changed to a sed call and whole patch can be dropped.

```sh
sed -i 's/^#if XV$/ifdef XV/g' src/fbdev.c
```

Reply:
Maybe leave a comment explaining why it's needed too.

Reply:
OK. Thank you for your tips!